### PR TITLE
Update onboard-windows-multi-session-device.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected all references to WVD

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8723

File: microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md [hunk 1]
```diff
@@ -37,16 +37,16 @@ Familiarize yourself with the [considerations for non-persistent VDI](/microsoft
...
 
 > [!NOTE]
-> The placement and configuration of the VDI onboarding startup script on the WVD golden image configures it as a startup script that runs when the WVD starts. It's **not** recommended to onboard the actual WVD golden image. Another consideration is the method used to run the script. It should run as early in the startup/provisioning process as possible to reduce the time between the machine being available to receive sessions and the device onboarding to the service. Below scenarios 1 and 2 take this into account.
+> The placement and configuration of the VDI onboarding startup script on the AVD golden image configures it as a startup script that runs when the AVD starts. It's **not** recommended to onboard the actual AVD golden image. Another consideration is the method used to run the script. It should run as early in the startup/provisioning process as possible to reduce the time between the machine being available to receive sessions and the device onboarding to the service. Below scenarios 1 and 2 take this into account.
 
 ### Scenarios
 
-There are several ways to onboard a WVD host machine:
+There are several ways to onboard a AVD host machine:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md]:
```suggestion
There are several ways to onboard an AVD host machine:
```

File: microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ For more information, see [Onboard Windows devices using Configuration Manager](
 
 #### Tagging your machines when building your golden image
 
-As part of your onboarding, you may want to consider setting a machine tag to can differentiate WVD machines more easily in the Microsoft Security Center. For more information, see
+As part of your onboarding, you may want to consider setting a machine tag to can differentiate AVD machines more easily in the Microsoft Security Center. For more information, see
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md]:
```suggestion
As part of your onboarding, you may want to consider setting a machine tag to differentiate AVD machines more easily in the Microsoft Security Center. For more information, see
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by JohanFreelancer9** [comment]
@msbemba, you have missed one suggestion. Could you please also commit that? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**